Resources/ServerInfo/Rules.txt

```diff
@@ -61,39 +62,40 @@ If you have any questions about these rules, please use the admin help menu by h
+ - A medical doctor does not need insulated gloves, the Head of Personnel does not need a gun, and the Captain does not need an assault rifle. If you, in your given role, require an item that does not fall within your job’s usual parameters, have a valid reason to obtain and keep it.
     - Manufacturing weapons, bombs, or deadly poisons before you know of any reason you would need them is self-antagonistic behavior.
  - Hiding Traitor objective items or preemptively securing them with higher security than usually required or than would make logical sense violates our powergaming and roleplay guidelines.
     - Do not, [color=#ff0000]under any circumstances[/color], hide the nuclear fission explosive, authentication disk, or other sensitive items in an impossible to see/access location.
 
 [color=#a4885c]7.[/color] Follow end-of-round (EOR) rules:
  - Significant end-of-round grief (EORG) is not allowed. This includes attacking, destroying, polluting, and severely injuring without reason both at and on the way to Central Command.
-    - Explosives and strategies with capacity for high collateral damage should not be used on or around the evacuation shuttle after it arrives.
+    - Explosives and strategies with capacity for high collateral damage should not be used on or around the evacuation shuttle after it arrives at the station.
 
 [color=#a4885c]Command, Security, and Antagonist Rules:[/color]
 [color=#a4885c]C1.[/color] Follow Command and Security Guidelines:
  - Security and Command roles are held to a higher standard of roleplay, and are expected to know and perform their jobs to the best of their ability. These roles often heavily impact the round and require more careful and conscientious roleplay; if you cannot meet these requirements do not take these roles.
     - Both departments are required to read and follow Delta-V Space Law, Standard Operating Procedure, Alert Procedure, and Company Policy to the best of their ability.
     - Abandoning your role to go do whatever you want instead of managing your department or maintaining security, abusing your position or using it to make arbitrary or malicious choices to the detriment of the station, and being negligent to the point of harm is prohibited.
  - If you need to leave the round, please notify administrators via AHelp menu and notify your fellow crew via departmental radio. If you have the capability to do so, ensure that you return your items as necessary and seek cryostorage if you are not planning on returning.
- - Do not give up or trade away Traitor objective items, departmental/Command gear, and sensitive equipment without excellent reason.
+ - Do not give up or trade away Traitor objective items, departmental/Command tools and remotes, and sensitive equipment without excellent reason.
```

Comment:
departmental tools = ce isnt allowed to give crowbars to people?

Reply:
More like Epi shouldnt be giving away use of their protowhatever, or cargo shouldnt give away PKAs or Salvage Suits.

A crowbar is part of the maints department anyway

Reply:
that only applies to command so scientists themselves are allowed to do it

this should just be in SOP that protolathes and circuit imprinters are controlled items so they are ic guilty of black marketeering if they give it away

Reply:
Its a "stopgap" until SOP is re-written anyhow. Just stops command giving away things for free for seemingly no reason. Should also help metafriending